src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.ps1

```diff
@@ -21,7 +21,7 @@ function Test-ValidateDeployment
 	# Setup
 	$rgname = Get-ResourceGroupName
 	$rname = Get-ResourceName
-	$rglocation = "CentralUSEUAP"
+	$rglocation = "centralus"
```

Comment:
Because when we use CentralUSEUAP rglocation we will failed to pass the deployment test case and throw 'Provided resource group does not exist' error message. So I searched another rglocation.